app/components/address-mapper.tsx

```diff
@@ -114,12 +108,14 @@ const AddressMapper: React.FC<AddressMapperProps> = ({
               isHazardDataLoading={isHazardDataLoading}
             />
           </Box>
-          <Map
-            coordinates={coordinates}
-            softStoryData={softStoryData}
-            tsunamiData={tsunamiData}
-            liquefactionData={liquefactionData}
-          />
+          <Suspense>
```

Comment:
Wrapping w/`<Suspense>` appears to be the correct for the prerender errors we were seeing on eg Next 14.